src/main/java/de/jetwick/snacktory/ArticleTextExtractor.java

```diff
@@ -1777,6 +1777,24 @@ protected String extractAuthorName(Document doc) {
                     System.out.println("AUTHOR: [class=kasten_titel]");
             }
 
+            // http://www.einnews.com/pr_news/339534444/rackspace-reaches-openstack-leadership-milestone-six-years-and-one-billion-server-hours
+            if (authorName.isEmpty()) {
```

Comment:
Probably outside the scope of this PR, but it would be nice to start moving these field specific rules to separate classes / files.